warehouse/ingest-core/src/main/java/datawave/ingest/data/config/ingest/BaseIngestHelper.java

```diff
@@ -14,6 +14,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import org.apache.accumulo.core.util.Pair;
```

Comment:
I thought we were trying to avoid using this class, maybe replace with apache common's Pair?